data/core/src/commonMain/kotlin/com/tunjid/heron/data/utilities/multipleEntitysaver/SavePostView.kt

```diff
@@ -63,6 +79,18 @@ internal fun MultipleEntitySaver.add(
                 embedEntity = embedEntity,
             )
         }
+        // handle fetch embed records in quoted post
+        postView.getEmbeddedFeedGenerator()?.let { generatorView ->
```

Comment:
The entities should not be added directly like this, you will miss edge cases. This does not save the `FeedGenerator` creator for example. internal methods already exist for all these types:

* To save feed generators, use [this](https://github.com/tunjid/heron/blob/main/data/core/src/commonMain/kotlin/com/tunjid/heron/data/utilities/multipleEntitysaver/SaveFeedGeneratorView.kt).
* To save lists use [this](https://github.com/tunjid/heron/blob/main/data/core/src/commonMain/kotlin/com/tunjid/heron/data/utilities/multipleEntitysaver/SaveListView.kt).
* To save starter packs, use [this](https://github.com/tunjid/heron/blob/main/data/core/src/commonMain/kotlin/com/tunjid/heron/data/utilities/multipleEntitysaver/SaveStarterPackView.kt).

There's no need to refine the logic already present there.